extension/main.js

```diff
@@ -23,10 +23,10 @@
 			if (interval !== null && interval !== parseInt(window.GitHubNotify.settings.get('interval'), 10)) {
 				window.GitHubNotify.settings.set('interval', interval);
```

Comment:
Seems like you should check for `interval` not being a number here (or even before this if block). I'd then set `interval` to `60` if it's not a number. This would keep `interval` from getting saved as an invalid value.